lib/assignment/widgets/assignments_screen.dart

```diff
@@ -132,24 +133,28 @@ class _AssignmentsScreenState extends State<AssignmentsScreen>
   }
 }
 
+typedef OnCourseClicked = void Function(Id<Course> courseId);
```

Comment:
Hmm. It doesn't really sound like a type. What about `CourseClickedCallback`?
Also, alliterations are fun.